src/tools/dev/scripts/bv_support/bv_adios.sh

```diff
@@ -74,7 +69,6 @@ function bv_adios_info
     export ADIOS_VERSION=${ADIOS_VERSION:-"1.13.1"}
     export ADIOS_FILE=${ADIOS_FILE:-"adios-${ADIOS_VERSION}.tar.gz"}
     export ADIOS_COMPATIBILITY_VERSION=${ADIOS_COMPATIBILITY_VERSION:-"${ADIOS_VERSION}"}
-    export ADIOS_URL=${ADIOS_URL:-"http://users.nccs.gov/~pnorbert"}
```

Comment:
I know we said we wanted to remove these URLs but having them around is useful, if only for reference. Sometimes I wonder where I should go to get a new version of a lib and that info is right in the bv script. Maybe these live on as comments?

Reply:
Problem --- this URL is no longer the right URL for Adios. 

So even a comment would be noise.

Reply:
Case by case basis.